modules/lm-coursier/src/main/scala/lmcoursier/CoursierConfiguration.scala

```diff
@@ -65,9 +73,51 @@ import xsbti.Logger
     withFollowHttpToHttpsRedirections(Some(true))
   def withStrict(strict: Strict): CoursierConfiguration =
     withStrict(Some(strict))
+  def withTtl(ttl: Duration): CoursierConfiguration =
+    withTtl(Some(ttl))
 }
 
 object CoursierConfiguration {
+  @deprecated("additional arguments exist now", "2020-03-12") // this is for binary compatibility with 2.0.0-RC3-4.
+  def apply(
```

Comment:
I initially didn't really grok data-class (did after pushing), but suspected something was supposed to happen. Still, MIMA felt like yelling at me. Will try again to avoid it

Reply:
Mmhh, MIMA shouldn't yell, thanks to the ` @since` annotation you added before the new fields. Let it yell on the CI here if it really does, I'll investigate.

Reply:
Mmhh, MIMA shouldn't yell, thanks to the ` @since` annotation you added before the new fields. Let it yell on the CI here if it really does, I'll investigate.

Reply:
removed the overload and did just that. Let's see how MIMA fares this time.. And I'll leave that to you

Reply:
there it is https://travis-ci.org/github/coursier/sbt-coursier/jobs/661602961?utm_medium=notification&utm_source=github_status

Reply:
ok, done.